catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DuplicateCommands.java

```diff
@@ -138,20 +152,37 @@ protected void duplicateInBatches(CatalogFacade queryFacade, CatalogFacade inges
                     rejectedExecutionHandler);
```

Comment:
❓ There should be a call to `StandardThreadFactoryBuilder.newThreadFactory("duplicateCommandsThread")` here. I'm marking it as a question because I don't see how it could be missing from this PR but not showing up as a diff here. A refactor with master should fix it.